feflow/settings/nonequilibrium_cycling.py

```diff
@@ -49,26 +52,21 @@ class NonEquilibriumCyclingSettings(Settings):
     class Config:
         arbitrary_types_allowed = True
 
-    # System Settings (from openfe)
-    system_settings: SystemSettings
     forcefield_cache: Optional[str] = (
         "db.json"  # TODO: Remove once it has been integrated with openfe settings
     )
 
     # Solvation settings
-    solvation_settings: SolvationSettings
+    solvation_settings: OpenMMSolvationSettings
 
     # Lambda settings
     lambda_functions = DEFAULT_ALCHEMICAL_FUNCTIONS
 
     # alchemical settings
-    alchemical_settings: AlchemicalSettings
+    alchemical_settings: AlchemicalSettings = AlchemicalSettings(softcore_LJ="gapsys")
 
-    # NEQ integration settings
-    timestep = 4.0 * unit.femtoseconds
-    neq_splitting = "V R H O R V"
-    eq_steps = 250000
-    neq_steps = 250000
+    # integrator settings
+    integrator_settings: PeriodicNonequilibriumIntegratorSettings
 
     # platform and serialization
```

Comment:
In the next iteration of this work we should switch this to use the OutputSettings and EngineSettings we have elsewhere, that way the user API is more consistent.

@dotsdl how big of a deal would it be if we did this in the next release? (i.e. would we need to do some kind of migration?)

Reply:
One thing with the `OutputSettings` is that the current state is pretty oriented towards the repex protocol, and the use of the `openmmtools.multistate.MultiStateSampler`, so it's not that transferable to the Neq Cycling protocol. We probably need to write some NEq sampler on `openmmtools`  that hopefully shares some API with the `MultiStateSampler` but this seems like a non-trivial amount of work.

Using the `OpenMMEngineSettings` should be already possible. I'll make the changes for that.

Reply:
I'll have a think - I think we can do make it so that the "look and feel" is the same, even if the two protocols differ in terms of base classes.